### PR TITLE
fix: Set the default value to false for the checkbox custom attribute

### PR DESCRIPTION
**Pull request by karthi07**
https://github.com/chatwoot/chatwoot/issues/7503

the default value of the custom attribute is always null, so I updated the default to false if the type is a checkbox.

### CODE REVIEW

**Comment by sojan-official** [comment]
@karthi07 Thank you so much for your contribution. We truly appreciate your efforts. 🙏 

Our team will prioritize reviewing and testing your changes according to our existing schedule and workload. 
Should there be a need for any further actions or information from your side, we'll follow up here. 📨

Once again, we're grateful for your involvement and contribution to our community. 🚀🌟 🙌  \

File: app/javascript/dashboard/routes/dashboard/conversation/customAttributes/CustomAttributeSelector.vue [hunk 1]
```diff
@@ -58,21 +58,27 @@ export default {
...
     async addAttribute(attribute) {
       try {
-        const { attribute_key } = attribute;
+        const {
+          attribute_key,
+          attribute_display_type,
+          default_value,
+        } = attribute;
+        const defaultValue =
+          attribute_display_type === 'checkbox' ? false : default_value || null;
```
Comment (muhsin-k) [inline app/javascript/dashboard/routes/dashboard/conversation/customAttributes/CustomAttributeSelector.vue:67]:
```suggestion
      const isCheckbox = attribute_display_type === 'checkbox';
      const defaultValue = isCheckbox ? false : default_value || null;
```

**Review by muhsin-k — APPROVED** [review]

**Comment by karthi07** [comment]
@muhsin-k  I'm not sure why the build is failing, any idea on how to resolve this?

**Comment by karthi07** [comment]
@muhsin-k resolved the spacing issue, this PR can be merged now, can you merge this PR ?

**Comment by karthi07** [comment]
@muhsin-k  can you provide any suggestions to resolve this failing pipeline?

**Comment by muhsin-k** [comment]
@karthi07 Apologies for the delayed response. The spec failures mentioned are not related to this pull request. We will address the spec issues separately and merge this pull request soon.

**Comment by sojan-official** [comment]
@muhsin-k  Ping

**Comment by pranavrajs** [comment]
@karthi07 Thanks for the PR, this was helpful.

**Pull request merged**